lib/observer/known_locations.ex

```diff
@@ -0,0 +1,3 @@
+defmodule Observer.KnownLocations do
```

Comment:
Added known locations module for now. Want to move this into a file in the future, but for now I just wanted to get it out of the main satellite.ex module.